Copyedits in "Operating System Regional Data Display" (#2134)
MicrosoftDocs/edge-developer: microsoft-edge/web-platform/os-regional-settings.md

In general, the website's internationalization user experiences are only as good as the regional code that they're able to determine for the current user. 

A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  For more information, see [ICU-TC Home Page](https://icu.unicode.org/home).

> To set the user's language preferences, Microsoft Edge gets the default values from the OS when it is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.

Reviewer: pls move the new "Preferred languages" screenshot to near here (it's mentioned in this sentence & table, maybe elsewhere too), and restore the "OS regional" screenshot.  Note that in the UI, "Preferred lang" is at top of UI and "OS region" is at bottom.  I looked at them both at once onscreen in the UI but they are spread too far apart to fit into a single capture at 100% zoom level.  They are not adjacent in the UI else single capture would be perfect.

Author: done

Author: We're moving into styleist changes, but yes I did this one.

| Language or region data | User impact |
|--------|-------------------------|

> | **Preferred language** in the **setting** page | The language selected for rendering Microsoft Edge. This language/region is also used for formatting values such as date, time, and numbers. |

Reviewer: ```suggestion
| **Preferred languages** in the **Settings** > **Languages** page | The language selected for rendering Microsoft Edge. This language/region is also used for formatting values such as date, time, and numbers. |
```
Check UI labels.  On my screen, has an 's' and the list is "English (United States)" and then below it, "English".  AFAIK, these are my defaults.

> |  `accept-language` HTTP header | A list of regional codes (as defined in preferred languages) sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |

Reviewer: the actual UI text, which should somehow be shown in bold here:
The "**Settings**" page > "**Languages**" page (selected on left) opens the Languages page within Settings (url is edge://settings/languages ) > the "**Preferred languages**" dropdown list, or dropdown menu.
```suggestion
|  `accept-language` HTTP header | A list of regional codes (as defined in **Preferred languages**) sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
```
![image](https://user-images.githubusercontent.com/45407486/185691348-f36cc9ed-fbd4-4438-bebd-d8fbf544f40f.png)

Reviewer: I really think another screenshot might be needed, like pasted into my previous comment, showing **Languages** nav on left, and the **Preferred languages** section.

Author: I just added a new image

> | `navigator.language` API (and similar) | Code inserted into a website that allows the browsers to display the correct regional formatting (as defined in preferred languages) for user input and other content produced locally. |

Reviewer: ```suggestion
| `navigator.language` API (and similar) | Code inserted into a website that allows the browsers to display the correct regional formatting (as defined in  **Settings** > **Languages** > **Preferred languages**) for user input and other content produced locally. |
```
